lib/cron.ts

```diff
@@ -28,6 +29,8 @@ export async function cron() {
   let errors: any[] = [];
 
   for (let i = lastCheckedId + 1; i <= latestPostId; i++) {
+    if (await checkIfPostWasChecked(i)) continue; // avoid double checking posts
```

Comment:
I think we could have moved the `await setLastCheckedId(latestPostId);` to here to avoid the extra storage?